ui/ui_vfinal/addon_strings_js.html

```diff
@@ -227,7 +227,7 @@
       'correlation': {
         'sidebarTitle': 'Correlation',
         'intentName': 'Correlation',
-        'description': 'Describes the degree of relationship between two values',
+        'description': 'Calculates the correlation between the two variables provided. Remember that the correlation ranges from -1 to 1',
```

Comment:
Instead of "Remember that", we can just say "Note that".

Reply:
Changed